tests/init/nested/test.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartTest "Nested"
+        rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "pushd $tmp"
+
+        # Setup
+        rlRun -s "tmt init"
+        rlAssertExists ".fmf/version"
+        rlAssertGrep "Tree .* initialized." "$rlRun_LOG"
+        rlRun "mkdir $tmp/foo && cd $tmp/foo"
+
+        # Test option --force
```

Comment:
Hint: Instead of separating test phases by mere comments I'd suggest using `beakerlib` phases which are intended exactly for this purpose:

```python
    rlPhaseStartTest "Test option --force"
        rlRun -s "tmt init --force" 0 "Test tmt init with option --force"
        rlAssertExists "$tmp/foo/.fmf/version"
        rlAssertGrep "Path '$tmp/foo' already has a parent root '$tmp'" "$rlRun_LOG"
        rlAssertGrep "Tree '$tmp/foo' initialized." "$rlRun_LOG"
        rlRun "rm -rf $tmp/foo/.fmf"
    rlPhaseEnd
```
Here the test is simple so it's not that crucial but it substantialy improves readability for more complex tests and speeds up reviewing test failures. Instead of long list of asserts you get a nice summary for each phase:

```
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
::   Test option --force
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

:: [ 09:40:51 ] :: [  BEGIN   ] :: Test tmt init with option --force :: actually running 'tmt init --force'
Path '/tmp/tmp.C7TUZshaTE/foo' already has a parent root '/tmp/tmp.C7TUZshaTE'.
Tree '/tmp/tmp.C7TUZshaTE/foo' initialized.
To populate it with example content, use --template with mini, base or full.
:: [ 09:40:52 ] :: [   PASS   ] :: Test tmt init with option --force (Expected 0, got 0)
:: [ 09:40:52 ] :: [   PASS   ] :: File /tmp/tmp.C7TUZshaTE/foo/.fmf/version should exist 
:: [ 09:40:52 ] :: [   PASS   ] :: File '/var/tmp/rlRun_LOG.p8yWAjnt' should contain 'Path '/tmp/tmp.C7TUZshaTE/foo' already has a parent root '/tmp/tmp.C7TUZshaTE'' 
:: [ 09:40:52 ] :: [   PASS   ] :: File '/var/tmp/rlRun_LOG.p8yWAjnt' should contain 'Tree '/tmp/tmp.C7TUZshaTE/foo' initialized.' 
:: [ 09:40:52 ] :: [  BEGIN   ] :: Running 'rm -rf /tmp/tmp.C7TUZshaTE/foo/.fmf'
:: [ 09:40:52 ] :: [   PASS   ] :: Command 'rm -rf /tmp/tmp.C7TUZshaTE/foo/.fmf' (Expected 0, got 0)
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
::   Duration: 1s
::   Assertions: 5 good, 0 bad
::   RESULT: PASS (Test option --force)
```

Reply:
Hi @psss, thanks for your suggestion. I have updated the test script to use `beakerlib phases`.

Reply:
Nice! Thanks.